README.md

```diff
@@ -275,11 +275,13 @@ print(3 / 0)! // undefined
 
 ## Strings
 
-Strings can be declared with single quotes or double quotes.
+Strings can be declared with single quotes, double quotes, backticks, or french quotes.
```

Comment:
What about german quotes `„` and `“`? Why do you call Guillemets „french quotes“?

Reply:
Did you know that in Austria these are reverted, basically `const const name = »antoine«!`?

Reply:
Yes, see https://github.com/TodePond/WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite/pull/228#discussion_r1316325715